Doc/library/constants.rst

```diff
@@ -79,6 +79,8 @@ A small number of constants live in the built-in namespace.  They are:
    :exc:`SyntaxError`), so they can be considered "true" constants.
 
 
+.. _site-consts:
```

Comment:
The objects above are all non-callable singleton or doubleton instances of their class.  (__debug__ is one of False or True).  All of the below are callable singleton or doubleton instances of custom classes in _sitebuiltins.  I would prefer that 'constants' be replaced by 'objects'/'callables'/'functions'/???, but this is secondary to the issue.

Reply:
I agree with your observation, they are not exactly constants. As the title of this section is "Built-in Constants", I won't make any changes, I feel it's a bit out of scope.

Reply:
I would take the opportunity to improve this to say that _the site module adds several globals to the built-in namespace_